docs_generator/Scanner.cs

```diff
@@ -110,6 +110,15 @@ private static string GenerateFileForMethods(List<MethodInfo> methods, IEnumerab
     {
         return string.Join("\n\n", methods.Select(method => GenerateTextForMethod(method, complexTypes)));
     }
+    
+    private static string FormatExampleBasedOnNewlines(string example)
+    {
+        if (!example.Contains('\n'))
+        {
+            return $"  - `{example}`";
+        }
+        return $"  - ```\n{string.Join("\n", example.Split("\n").Select(line => $"    {line}"))}\n    ```";
```

Comment:
Change this to a ternary and invert the condition?

Untested, unformatted code:
```suggestion
return example.Contains('\n')) 
    ? $"  - ```\n{string.Join("\n", example.Split("\n").Select(line => $"    {line}"))}\n    ```"
    : $"  - `{example}`";
```

Reply:
I usually avoid them, since I find them difficult to read, but hey, totally fine to apply them if they're requested!

Added it in 6e837aa, but had to resolve an issue in 2acd6ea.  
Also rebased the PR as #440 got integrated first. (Thanks for both reviews! ❤️)